src/rad/resources/schemas/wfi_wcs-1.0.0.yaml

```
@@ -0,0 +1,75 @@
+%YAML 1.1
+---
+$schema: asdf://stsci.edu/datamodels/roman/schemas/rad_schema-1.0.0
+id: asdf://stsci.edu/datamodels/roman/schemas/wfi_wcs-1.0.0
+
+title: Level 2 (L2) WCS information for Roman Wide Field Instrument (WFI) Rate Image.
+
+datamodel_name: WfiWcsModel
+archive_meta: None
+
+type: object
+properties:
+  meta:
+    allOf:
+      - $ref: asdf://stsci.edu/datamodels/roman/schemas/basic-1.0.0
+      - type: object
+        properties:
+          coordinates:
+            title: Name Of The Coordinate Reference Frame
+            tag: asdf://stsci.edu/datamodels/roman/tags/coordinates-1.0.0
+          ephemeris:
+            title: Ephemeris Data Information
+            tag: asdf://stsci.edu/datamodels/roman/tags/ephemeris-1.0.0
+          exposure:
+            title: Exposure Information
+            tag: asdf://stsci.edu/datamodels/roman/tags/exposure-1.0.0
+          instrument:
+            title: WFI Observing Configuration
+            tag: asdf://stsci.edu/datamodels/roman/tags/wfi_mode-1.0.0
+          observation:
+            title: Observation Identifiers
+            tag: asdf://stsci.edu/datamodels/roman/tags/observation-1.0.0
+          pointing:
+            title: Spacecraft Pointing Information
+            tag: asdf://stsci.edu/datamodels/roman/tags/pointing-1.0.0
+          program:
+            title: Program Information
+            tag: asdf://stsci.edu/datamodels/roman/tags/program-1.0.0
+          velocity_aberration:
+            title: Velocity Aberration Correction Information
+            tag: asdf://stsci.edu/datamodels/roman/tags/velocity_aberration-1.0.0
+          visit:
+            title: Visit Information
+            tag: asdf://stsci.edu/datamodels/roman/tags/visit-1.0.0
+          wcsinfo:
+            title: World Coordinate System (WCS) Parameters
+            tag: asdf://stsci.edu/datamodels/roman/tags/wcsinfo-1.0.0
+        required: [coordinates, ephemeris, exposure,
+                   instrument, observation, pointing, program,
+                   velocity_aberration, visit, wcsinfo]
+      - properties:
+          wcs_fit_results:
+            title: tweakreg/GAIA fit results
+            description: |
+              Results from the tweakreg result when aligning to GAIA.
+              If missing or None, GAIA alignment was not done or failed.
+            type: object
```

```suggestion
          wcs_fit_results:
            title: tweakreg/GAIA fit results
            description: |
              Results from the tweakreg result when aligning to GAIA.
              If missing or None, GAIA alignment was not done or failed.
            type: object
        required: [coordinates, ephemeris, exposure,
                   instrument, observation, pointing, program,
                   velocity_aberration, visit, wcsinfo]
```
This will move this object with the rest on the block, making it easier to parse. I tested locally against [RDM 477](https://github.com/spacetelescope/roman_datamodels/pull/477), and it passed.